notifications_utils/formatters.py

```diff
@@ -619,8 +624,73 @@ def add_language_divs(_content: str) -> str:
 def remove_language_divs(_content: str) -> str:
     """Remove the tags from content. This fn is for use in the email
     preheader, since this is plain text not html"""
-    content = re.compile(FR_OPEN).sub("", _content)
-    content = re.compile(FR_CLOSE).sub("", content)
-    content = re.compile(EN_OPEN).sub("", content)
-    content = re.compile(EN_CLOSE).sub("", content)
+    return remove_tags(_content, FR_OPEN, FR_CLOSE, EN_OPEN, EN_CLOSE)
+
+
+def add_img_tag(_content: str, tag, img_location, alt_text="", height=300, width=300) -> str:
+    """
+    Custom parser to add custom img in the email.
+
+    This is a custom temporary change not meant to exist for more than a few
+    weeks. This should either be removed or upgraded into a full-fledged
+    feature.
+
+    TODO: Review, remove/upgrade this functionality.
+    """
+    ga_seal_regex = re.compile(f"{tag}")  # matches [[tag]]
+    content = ga_seal_regex.sub(
+        r"""<div style="margin: 20px auto 30px auto;">
+          <img
+            src="{img_loc}"
+            alt="{alt}"
+            height="{h}"
+            width="{w}"
+          />
+        </div>""".format(
+            img_loc=img_location, alt=alt_text, h=str(height), w=str(width)
+        ),
+        _content,
+    )
+
+    return content
+
+
+def add_ircc_ga_seal(_content: str) -> str:
+    """
+    Custom parser to add IRCC Global Affairs seal logo.
+
+    This is a custom temporary change not meant to exist for more than a few
+    weeks. This should either be removed or upgraded into a full-fledged
+    feature.
+
+    TODO: Review, remove/upgrade this functionality.
+    """
+    img_loc = "https://assets.notification.canada.ca/gc-ircc-ga-seal.png"
+    alt_text = "Global Affairs Canada / Affaires mondiales Canada"
+    return add_img_tag(_content, TAG_IMG_IRCC_GLOBAL_AFFAIRS, img_loc, alt_text, 339, 322)
+
+
+def add_ircc_coat_of_arms(_content: str) -> str:
+    """
+    Custom parser to add IRCC coat of arms logo.
+
+    This is a custom temporary change not meant to exist for more than a few
+    weeks. This should either be removed or upgraded into a full-fledged
+    feature.
+
+    TODO: Review, remove/upgrade this functionality.
+    """
+    img_loc = "https://assets.notification.canada.ca/gc-ircc-coat-of-arms.png"
```

Comment:
This URL is returning an "Access denied" error for me.

Reply:
Working for me now.